mozilla/blurts-server
# MNTOR-2286/show correct in-progress scan results

# References: 
Jira: MNTOR-2286




# Description

This is a temporary approach - it turns out the scan ID is reused, so we need to account for that.

This needs testing


# Screenshot (if applicable)

Not applicable.

# How to test

You need to be able to make yourself a subscriber - I've included a partially-finished tool (working on finishing that) to do this. Note that you need to put you primary email address in `src/app/(proper_react)/redesign/(authenticated)/admin/users/page.tsx` to make it work.

# Checklist (Definition of Done)
